### DISCUSSION TITLE
Can I use this integration with a EVCC installation as an HA Add-on?

### DISCUSSION THREAD

**Discussion by ddeconin-gh** [marq24/ha-evcc#8] [Q&A]
In that case, should the URL just be the HA URL with the EVCC port?

**Comment by marq24** [comment]
What URL you enter in your browser to open the local EVCC website? - if it's the same IP then your HA installation - but instead of 8123 the port, 7070 - then "yes" - just enter http://[YOUR-IP-HERE]:7070

**Reply by Sammy521** [reply]
That workes for me. But since a few days it does not work anymore. Maybe since any update or something.

Even when I delete the integration konfig and add a new one it does not work anymore/again:

<img width="739" alt="grafik" src="https://github.com/user-attachments/assets/fe929673-9d94-4011-aaeb-f1164f540781" />

Does anybody know what to do? 
Today I updated HA to 25.1 but that did not change a thing either. 
EVCC is V 0.132.1

**Reply by marq24** [reply]
update the integration...

**Reply by Sammy521** [reply]
how to? normally the systems tells me that there is an update but I dont get any info this time / for that integration.

It is: 2024.12.2

EDIT: Ok dont know how but after open the integration store and searched again for evcc I got the update. Now it works again. Thanks :)

**Comment by matthewbayard** [comment]
How did you get on? I am having the same issue too. I can get the data to pull on windows but cant get the data in HA.

**Reply by marq24** [reply]
same question to you, what URL you use to 'get the data to pull on windows' ?
